br/pkg/utils/misc.go

```diff
@@ -46,8 +46,7 @@ const (
 	// Also note that the offline threshold in PD is 20s, see
 	// https://github.com/tikv/pd/blob/c40e319f50822678cda71ae62ee2fd70a9cac010/pkg/core/store.go#L523
 
-	// After talk to PD members 100s is not a safe number. set it to 600s
-	storeDisconnectionDuration = 600 * time.Second
+	storeDisconnectionDuration = 100 * time.Second
```

Comment:
It's safe.
The only use case is in main backup loop right now.
It required us detected disconnect tikv store immediately. if set 10mins(600s). the integration test case: `br_tikv_outage` would take a long time to finish. because we always retry to connect a outaged tikv. 

with a small value. the price is try more backup rounds.